config/samples/bpfman.io_v1alpha1_bpfapplication.yaml

```diff
@@ -9,50 +9,113 @@ spec:
   nodeselector: {}
   bytecode:
     image:
-      url: quay.io/bpfman-bytecode/go-app-counter:latest
+      url: quay.io/bpfman-bytecode/app-test:latest
+      # imagepullpolicy: Always
   programs:
-    - type: Kprobe
+    - bpffunctionname: kprobe_test
```

Comment:
I really don't like `bpffunctioname`. It should either be `bpfFunctionName` or just `functionName` since we're in a `Bpf` CRD.. or hey, even just `function`

Reply:
We made it "bpffunctionname" after much discussion a while back when we started using the bpf function name instead of a name defined in the section declaration.  We also wanted to disambiguate this function name from the name of the function that some bpf programs attach to.

That said, I don't like it either :-).  I changed "bpfunctionname" to just "name", and the "func_name" and "function_name" to just "function".